cmd/end2endtest/helpers.go

```diff
@@ -508,11 +507,13 @@ func ballotVotes(b ballotData, nvotes int) ([][]int, [][]*types.BigInt) {
 
 // sendVotes sends a batch of votes concurrently
 // (number of goroutines defined in t.config.parallelCount)
-func (t *e2eElection) sendVotes(votes []*apiclient.VoteData) (errs map[int]error) {
-	errs = make(map[int]error)
```

Comment:
the point of errs being a map was for the caller to be able to see which vote produced which error.

why are you discarding that feature?

Reply:
I didn't see the usefulness before, due is possible that the votes will not keep the order, to look for it when we are debugging. But, maybe I am not seeing the approach correctly.